src/System.Diagnostics.Process/tests/ProcessStandardConsoleTests.cs

```diff
@@ -12,7 +12,7 @@ public class ProcessStandardConsoleTests : ProcessTestBase
     {
         private const int s_ConsoleEncoding = 437;
 
-        [Fact]
+        [ConditionalFact(nameof(PlatformDetection)+"."+nameof(PlatformDetection.IsNotWindowsNanoServer))]
```

Comment:
VS must have auto formatted the other files when opened. Interpolated strings wont work here, as the attribute should have constant values as arguments, it cannot be expressions..